pkg/istio/common/common_test.go

```diff
@@ -0,0 +1,47 @@
+package common
```

Comment:
Can we move the TestMemberToSpiffe, TestMemberToOriginJwtSubject, TestParseAssertionEffect, TestParseAssertionAction, and TestParseAssertionResource tests to this file as well?